src/components/Hoverable/ActiveHoverable.tsx

```diff
@@ -0,0 +1,152 @@
+import {cloneElement, forwardRef, MutableRefObject, Ref, RefCallback, useCallback, useEffect, useImperativeHandle, useMemo, useRef, useState} from 'react';
+import {DeviceEventEmitter} from 'react-native';
+import CONST from '@src/CONST';
+import HoverableProps from './types';
+
+/**
+ * Assigns a ref to an element, either by setting the current property of the ref object or by calling the ref function
```

Comment:
A newline after the description is conventional in Expensify, I think. It seems that _most_ of the time it's there.